blog/themes/3mdeb/layouts/partials/widget-tags.html

```diff
@@ -5,6 +5,7 @@ <h1 class="title is-5">Tags</h1>
         {{ range $name, $taxonomy := .Site.Taxonomies.tags }}
             <span class="tag"><a href="{{ "tags" | absURL }}/{{ $name | urlize }}">{{ $name }}</a></span>
         {{ end }}
+        <a href="" class="read-more">Read More</a>
```

Comment:
@artur-rs failed to add pagination here, I quess the best way is to show first 10 items then use Read more button to increase it, here i found a tip of infinitive scrolling pagination: https://mikeroibu.com/post/hugo-infinite-scrolling/
btw. would work on posts one